Compile error with go 1.12

Getting this error after upgrading to go 1.12, Mac OS Mojave.

`github.com/jbowtie/gokogiri/xml/document.go:330:19: identifier "_Ctype_struct__xmlDoc" may conflict with identifiers generated by cgo`

Fails with this error on build.

--- a/xml/document.go
+++ b/xml/document.go
@@ -327,7 +327,7 @@ func (document *XmlDocument) CreateTextNode(data string) (text *TextNode) {
 	dataPtr := unsafe.Pointer(&dataBytes[0])
 	nodePtr := C.xmlNewText((*C.xmlChar)(dataPtr))
 	if nodePtr != nil {
-		nodePtr.doc = (*_Ctype_struct__xmlDoc)(document.DocPtr())
+		nodePtr.doc = (*C.xmlDoc)(document.DocPtr())
 		text = NewNode(unsafe.Pointer(nodePtr), document).(*TextNode)
 	}
 	return

